samples/exposed-migration/README.md

```diff
@@ -0,0 +1,36 @@
+# Exposed Migration
+
+This sample project illustrates how to:
+
+- generate a migration script using Exposed
+- apply a migration using a third-party library like Flyway with a migration script generated by Exposed
+
+## Generating a migration script
+
+In `Tables.kt`, the table `UsersV1` represents the first version of the Users table, which does not have a primary key.
+The table `UsersV2` represents what `UsersV1` would look like if it is modified to have a primary key. You do not need
+to create new versions of your existing tables in your project. It is done this way in this sample project for
+demonstration purposes only. Differences between your modified Exposed tables and their corresponding definitions in the
```

Comment:
If there's a concern that this example isn't representative of a real-world use case, maybe this potential point of confusion could be reduced by simulating the existing database with just plain SQL. For example, by removing `UsersV1` and just using `exec()` in the first transaction block (or even with another sql script). `existingPrimaryKeys()` should still work with `UsersV2` (or you could show that an insertion of a previous id throws a constraint violation exception in a try-catch block).

Reply:
I agree; let's try doing this. We also can rename UsersV* to just Users